jsons/UnitPromotions.json

```diff
@@ -618,95 +618,100 @@
 		"prerequisites": ["Buffalo Chest II"],
 		"unitTypes": ["Melee","Gunpowder"]
 	},
-		{
+	{
 		"name": "Epic of Battle",
 		"uniques": ["[+15]% Strength <vs cities>"],
 	},
 	{
-	"name": "Sauna",
-	"uniques": ["+10 HP when healing"],
+		"name": "Sauna",
+		"uniques": ["+10 HP when healing"],
```

Comment:
Im pretty sure the +10 is also parameterized

Reply:
So it would be [+10]?

Reply:
I think so

Reply:
resolved in https://github.com/ravignir/RekMOD/pull/43/commits/a7f15d56c535c9faaec0fedf201e82619fcf33f5